README.md

```diff
@@ -35,30 +35,40 @@ Screenshot (will need to be updated as syntax coloring and UI polish is added):
 
 ![xi screenshot](/doc/img/xi-mac-screenshot.png?raw=true)
 
-
 ## Getting started
 
 ### Requirements
+
 - [Xcode 9.x](https://developer.apple.com/xcode/)
-- [Rust](https://www.rust-lang.org/). We test against the latest stable version,
-and recommend installing through [rustup](https://rustup.rs).
+- [Rust](https://www.rust-lang.org/). We test against the latest stable version, and recommend installing through [rustup](https://rustup.rs).
 
+### Installing
 
-Note: the front-end and back-end are split into two separate repositories. This
+*Note:* the front-end and back-end are split into two separate repositories. This
 is the front-end, and the back-end (or core) is now in
-[xi-editor](https://github.com/xi-editor/xi-editor). Make sure to have that checked out
-as a subdirectory.
+[xi-editor](https://github.com/xi-editor/xi-editor). It is contained in a submodule that is checked out during the clone command.
 
-```
-> git clone https://github.com/xi-editor/xi-mac
+clone the repository with:
+
+```bash
+> git clone --recurse-submodules https://github.com/xi-editor/xi-mac
 > cd xi-mac
-> git clone https://github.com/xi-editor/xi-editor
+```
+
+and open with
```

Comment:
maybe this should be called 'build and open'? Or we could make these two steps?

Reply:
That's a much better description. Updating it now.